services/core/java/com/android/server/pm/PackageManagerService.java

```diff
@@ -703,6 +703,12 @@
 
     private static final String RANDOM_DIR_PREFIX = "~~";
 
+    /**
+     * The Google signature faked by microG.
```

Comment:
Instead of faked "by microG", can you say "for microG" to make it clear that it's not microG doing it?